plugins/by-name/snacks/default.nix

```
@@ -0,0 +1,205 @@
+          Icon for `trace` notifications.
+        '';
+      };
+      style =
+        defaultNullOpts.mkEnum
+          [
+            "compact"
+            "fancy"
+            "minimal"
+          ]
+          "compact"
+          ''
+            Style of notifications.
+          '';
+      top_down = defaultNullOpts.mkBool true ''
+        Whether to place notifications from top to bottom.
+      '';
+      date_format = defaultNullOpts.mkStr "%R" ''
+        Time format for notifications.
+      '';
+      refresh = defaultNullOpts.mkUnsignedInt 50 ''
+        Time in milliseconds to refresh notifications.
+      '';
+    };
+    quickfile = {
+      enabled = defaultNullOpts.mkBool true ''
+        Whether to enable `quickfile` plugin.
+      '';
+      exclude = defaultNullOpts.mkListOf types.str [ "latex" ] ''
+        Filetypes to exclude from `quickfile` plugin.
+      '';
+    };
+    statuscolumn = {
+      enabled = defaultNullOpts.mkBool true ''
+        Whether to enable `statuscolumn` plugin.
+      '';
+      left =
+        defaultNullOpts.mkListOf types.str
+          [
+            "mark"
+            "sign"
+          ]
+          ''
+            Priority of signs on the left (high to low).
+          '';
+      right =
+        defaultNullOpts.mkListOf types.str
+          [
+            "fold"
+            "git"
+          ]
+          ''
+            Priority of signs on the right (high to low)
+          '';
+      folds = {
+        open = defaultNullOpts.mkBool false ''
+          Whether to show open fold icons.
+        '';
+        git_hl = defaultNullOpts.mkBool false ''
+          Whether to use Git Signs hl for fold icons.
+        '';
+      };
+      git = {
+        patterns =
+          defaultNullOpts.mkListOf types.str
+            [
+              "GitSign"
+              "MiniDiffSign"
+            ]
+            ''
+              Patterns to match Git signs.
+            '';
+      };
+      refresh = defaultNullOpts.mkUnsignedInt 50 ''
+        Time in milliseconds to refresh statuscolumn.
+      '';
+    };
+    words = {
+      enabled = defaultNullOpts.mkBool true ''
+        Whether to enable `words` plugin.
+      '';
+      debounce = defaultNullOpts.mkUnsignedInt 200 ''
+        Time in ms to wait before updating.
+      '';
+      notify_jump = defaultNullOpts.mkBool false ''
+        Whether to show a notification when jumping.
+      '';
+      notify_end = defaultNullOpts.mkBool true ''
+        Whether to show a notification when reaching the end.
+      '';
+      foldopen = defaultNullOpts.mkBool true ''
+        Whether to open folds after jumping.
+      '';
+      jumplist = defaultNullOpts.mkBool true ''
+        Whether to set jump point before jumping.
+      '';
+      modes =
+        defaultNullOpts.mkListOf types.str
+          [
+            "n"
+            "i"
+            "c"
+          ]
+          ''
+            Modes to show references.
+          '';
+    };
+  };
+}
```

```suggestion
  };

  settingsExample = {
    ...
  };
}
```

Please, add a `settingsExample` entry with 5-10 options.
You can search github for user configs: `language:lua require('snacks').setup`.